lib/bugsnag/cleaner.rb

```diff
@@ -43,7 +43,13 @@ def traverse_object(obj, seen, scope)
       when String
         clean_string(obj)
       else
-        str = obj.to_s rescue RAISED
+        # guard against objects that raise or blow the stack when stringified
+        begin
+          str = obj.to_s rescue RAISED
```

Comment:
Does the `rescue RAISED` not catch all errors already?

Reply:
It catches all errors that inherit from `StandardError`, but `SystemStackError` doesn't so it won't be caught here without the separate `rescue`

I can make another `rescue` block for `StandardError` explicitly if it's confusing; something like:

```ruby
begin
  str = obj.to_s
rescue StandardError
  str = RAISED
rescue SystemStackError
  str = RECURSION
end
```

Reply:
Cool - yes, that would help (me!)

Reply:
Made the change in a1ad37e :)